### ISSUE TITLE
Inquire about the training details of Digits-DG on closed-set.

### ISSUE THREAD

**Issue by ZMC-IIIM** [zzwdx1234/MEDIC#4]
When training Digits-DG using Convnet as the backbone and a single classifier for prediction, while keeping other parameters unchanged, the training accuracy is similar to DeepALL in Digits-DG. I have tried adjusting the learning rate and batch size, but the results have not been satisfactory. Could you please provide some training details?

**Comment by zzwdx** [comment]
Please try setting the learning rate to 0.001 and the batch size to 32. Besides, in MEDIC, it's typically not recommended to train the close set classifier separately as it is often tied with the multi-binary classifier.
If you are still dissatisfied with the results, please consider trying our extended work [MEDIC++](https://github.com/zzwdx/MEDIC-plus) later, which employs a first order and multi-step strategy with faster execution speed and improved performance. Its close set accuracy has been validated on 5 datasets in DomainBed benchmark without using the multi-binary classifier.
